tools/depmod.c

```diff
@@ -1572,6 +1582,25 @@ static int depmod_load_modules(struct depmod *depmod)
 
 load_info:
 		kmod_module_get_info(mod->kmod, &mod->info_list);
```

Comment:
We can even save some memory, because the requirement of `mod->info_list` is gone. All its use cases can be covered with the newly introduced arrays now.

Reply:
Oh, I see... Too bad then :/